### PR TITLE
Resolv conf without systemd resolved

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: pyanaconda/installation.py [hunk 1]
```diff
@@ -392,8 +393,9 @@ def _prepare_installation(self, payload, ksdata):
 
         # Make name resolution work for rpm scripts in chroot.
         # Also make sure dns resolution works in %post scripts
-        # if systemd-resolved is not available.
-        if conf.system.provides_resolver_config:
+        # when systemd-resolved is not available.
+        if conf.system.provides_resolver_config and \
+                not is_service_installed("systemd-resolved.service"):
```
Comment (KKoukiou) [inline pyanaconda/installation.py:398]: Dont you need here, root=self._sysroot argument? You care about the target system service, no?

Comment (rvykydal) [reply pyanaconda/installation.py:398]: We care about installer environment service.
At this point the target system is empty and our only indication that systemd-resolved is not used on the system (and will not be installed) is that it is missing in the image ( https://github.com/rhinstaller/anaconda/pull/5488).

**Review by KKoukiou — APPROVED** [review]

File: pyanaconda/network.py [hunk 2]
```diff
@@ -214,6 +218,22 @@ def iface_for_host_ip(host_ip):
     return route_info[route_info.index("dev") + 1]
 
 
+def copy_resolv_conf_to_root(root="/"):
+    """Copy resolv.conf to a system root."""
+    src = "/etc/resolv.conf"
+    dst = os.path.join(root, src.lstrip('/'))
+    if not os.path.isfile(src):
+        log.debug("%s does not exist", src)
```
Comment (jkonecny12) [inline pyanaconda/network.py:226]: Maybe the debug messages could be more self explanatory? I'm not sure if I would understand why this information should be interesting to me from the message.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me otherwise.

File: pyanaconda/network.py (hunk 2 above)
Comment (M4rtinK) [reply pyanaconda/network.py:226]: Agreed, I think it should mention something about resolv conf in the error message. :)

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

File: pyanaconda/network.py (hunk 2 above)
Comment (rvykydal) [reply pyanaconda/network.py:226]: I didn't want to modify a revert. Also, the message says "/etc/resolv.conf does not exist" in the log in context of the "Copy resolv.conf to sysroot" task. Which is quite clear I think.

Comment (jkonecny12) [reply pyanaconda/network.py:226]: OK, sorry I missed it is a revert - my baf

**Pull request merged**